tests/functional/settings_clients.js

```diff
@@ -53,21 +52,55 @@ define([
       return this.remote.then(clearBrowserState());
     },
 
-    'device panel is not visible without query param': function () {
+    'sessions are listed in clients view': function () {
       return this.remote
-        .then(openPage(SIGNIN_URL, '#fxa-signin-header'))
+        .then(openPage(SIGNIN_URL + '?sessionsListVisible=1', '#fxa-signin-header'))
         .then(fillOutSignIn(email, FIRST_PASSWORD))
 
         .then(testElementExists('#fxa-settings-header'))
-        .then(noSuchElement('#devices'));
+        .then(click('#clients .settings-unit-stub button'))
+
+        // current session has the text `current session`
+        .then(FunctionalHelpers.testElementTextEquals(
+          '.client-webSession:nth-child(1) .client-name + .device-current',
+          'current session'
+        ))
+
+        // first session has the user agent
+        .then(testElementTextEquals(
+          '.client-webSession:nth-child(1) .client-name',
+          'Web Session, Firefox 40'
+        ))
+
+        // second session is the node.js session from test setup
+        .then(testElementTextEquals(
+          '.client-webSession:nth-child(2) .client-name',
+          'Web Session, node-XMLHttpRequest null'
```

Comment:
@vbudhram this is basically because node-XMLHttpRequest does not provide a version. Normal users will hopefully not see this :P